### DISCUSSION TITLE
Release of TomoATT v1.0.0

### DISCUSSION THREAD

**Discussion by xumi1993** [TomoATT/TomoATT#3] [Announcements]
@TomoATT/ntu-users @TomoATT/community-users 
We are pleased to announce the public release of [TomoATT](https://github.com/TomoATT/TomoATT.git) v1.0.0, our latest innovative software for advanced traveltime tomography. We cordially invite our users to share TomoATT with their peers, colleagues, and the broader research community.

**Comment by 1M-git** [comment]
[like]  Bai Yiming reacted to your message:
________________________________
From: Mijian Xu ***@***.***>
Sent: Thursday, November 21, 2024 3:13:40 PM
To: TomoATT/TomoATT ***@***.***>
Cc: Bai Yiming ***@***.***>; Team mention ***@***.***>
Subject: [TomoATT-Community] Release of TomoATT v1.0.0 (Discussion #3)


[Alert:
 Non-NTU Email] Be cautious before clicking any link or attachment.

@TomoATT/ntu-users<https://github.com/orgs/TomoATT/teams/ntu-users> @TomoATT/community-users<https://github.com/orgs/TomoATT/teams/community-users>
We are pleased to announce the public release of TomoATT<https://github.com/TomoATT/TomoATT.git> v1.0.0, our latest innovative software for advanced traveltime tomography. We cordially invite our users to share TomoATT with their peers, colleagues, and the broader research community.

—
Reply to this email directly, view it on GitHub<https://github.com/orgs/TomoATT/discussions/3>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AUEWNT45JRV3Z5YZWPTUKR32BX2CJAVCNFSM6AAAAABSHHLQYWVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGUZDONRXGQ>.
You are receiving this because you are on a team that was mentioned.Message ID: ***@***.***>

________________________________

CONFIDENTIALITY: This email is intended solely for the person(s) named and may be confidential and/or privileged. If you are not the intended recipient, please delete it, notify us and do not copy, use, or disclose its contents.
Towards a sustainable earth: Print only when necessary. Thank you.
